pytentiostat/config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "Experiment":{
```

Comment:
this would probably be better as ``experiment_type`` and the schema is ``"type": "str", "eallow" = ["LSV", "CV", "constant"]`` rather than a nested dictionary.  i.e., the user specifies one from a list of allowed values.